gufe/tests/conftest.py

```diff
@@ -67,18 +69,51 @@ def get_test_filename(filename):
 ]
 
 
-_pl_benchmark_url_pattern = (
-    "https://github.com/OpenFreeEnergy/openfe-benchmarks/blob/main/openfe_benchmarks/data/{name}.pdb?raw=true"
+_pl_benchmark_url_pattern = "https://github.com/OpenFreeEnergy/openfe-benchmarks/tree/f577a88d94b6deae1c0de7eb926edf48dd42b72d/openfe_benchmarks/data/{name}.pdb?raw=true"
```

Comment:
we don't need this anymore, correct?

Reply:
Correct -- We should either comment this line out OR remove all the bits that downloaded from github. Do you have a preference? I don't like to leave commented code around since it is confusing, but this also is testing code so I tried to explain it here:
https://github.com/OpenFreeEnergy/gufe/pull/553/files#diff-ef3697965a0e9abdb11db6defdf930637dbc635bf5ac13f9b022f7a88f536099R75-R76
and commented out here:
https://github.com/OpenFreeEnergy/gufe/pull/553/files#diff-ef3697965a0e9abdb11db6defdf930637dbc635bf5ac13f9b022f7a88f536099R110-R112
I am also happy to rip it all out as well